lib/galaxy/webapps/galaxy/services/history_contents.py

```diff
@@ -239,7 +239,7 @@ class CreateHistoryContentPayloadFromCollection(CreateHistoryContentPayloadFromC
         description="TODO",
     )
     copy_elements: Optional[bool] = Field(
-        default=False,
+        default=True,
         title="Copy Elements",
         description=(
             "If the source is a collection, whether to copy child HDAs into the target "
```

Comment:
Can you please change the comment on the following line accordingly?

Reply:
yes, rebased into the commit.

Reply:
Yep the docs really confused me here... thanks!